internal/app/user-api/routing/routes.go

```diff
@@ -0,0 +1,19 @@
+package routing
+
+import (
+	"github.com/afernandowex/ps-tag-onboarding-go/internal/app/user-api/controller"
+	"github.com/labstack/echo/v4"
+)
+
+type IRoutes interface {
+	InitializeRoutes(e *echo.Echo)
+}
+
+type Routes struct {
+	Controller controller.IUserController
```

Comment:
You should name this UserController, in the future you could have other controllers to base routes off.